content/millennium/r4/scheduling/appointment.md

```diff
@@ -249,6 +249,17 @@ Create a new Appointment.
 
     POST /Appointment
 
+_Implementation Notes_
+
+* The modifier elements [implicitRules] and [modifierExtension] are not supported and will be rejected if present.
+* `Appointment.status` must be set to `accepted`.
```

Comment:
It appears that we only accept a status of 'booked' for R4.

Reply:
Updated in this commit:
https://github.com/cerner/fhir.cerner.com/pull/311/commits/9dc3c856e57a3a3caffe0ea6727b0e4c272332e1